python/tests/integrations/langgraph_tests/test_langgraph_integration.py

```
@@ -86,6 +91,327 @@ def test_handler_type(callback_handler: NemoRelayCallbackHandler):
+    assert command_result["offset"] == 9
+    assert command_result["messages"][-1].content == "updated"
+    assert failure_result["messages"][-1].status == "error"
+    assert {event.name for event in subscribed_events if isinstance(event, nemo_relay.ScopeEvent)} == {
+        "request",
+        "update_offset",
+        "fail",
+    }
+
+
+@pytest.mark.parametrize("use_async", [False, True])
+def test_create_tool_node_preserves_selective_error_handling(use_async: bool):
```

Ruff (0.16.2)

[warning] Boolean-typed positional argument in function definition

(FBT001)